lib/ansible/modules/web_infrastructure/ansible_tower/tower_organization.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+#coding: utf-8 -*-
+
+# (c) 2016, Wayne Witzel III <wayne@riotousliving.com>
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: tower_organization
+version_added: "2.3"
+short_description: create, update, or destroy Ansible Tower organizations
+description:
+    - Create, update, or destroy Ansible Tower organizations. See
+      U(https://www.ansible.com/tower) for an overview.
+options:
+    name:
+      description:
+        - Name string to use for the organization.
+      required: True
+      default: null
+    description:
+      description:
+        - String to use for the organization.
+      required: False
+      default: null
+    state:
+      description:
+        - Desired state of the resource.
+      required: False
+      default: "present"
+      choices: ["present", "absent"]
+    config_file:
```

Comment:
host and verify_ssl are common settings in module auth, see other cloud/api modules for same.

we normally have auth at module level and optionally at config file/env  as options.